src/Sidebar/Sidebar.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import { Meta, Story } from "@storybook/react";
+import React from "react";
+
+import { SquareButton } from "../SquareButton";
+import { Decorator } from "../utils/Decorator";
+import * as fixtures from "./fixtures";
+import { Sidebar, SidebarProps } from "./Sidebar";
+
+const props: SidebarProps = {
+  active: "menu1",
+  menuItems: fixtures.menu,
+  onMenuItemClick: () => undefined,
+};
+
+export const Default: Story = () => <Sidebar {...props} />;
+export const WithToolbar: Story = () => (
+  <Sidebar {...props} toolbar={<SquareButton>tool</SquareButton>} />
+);
+
+export default {
+  title: "Sidebar",
+  decorators: [Decorator],
```

Comment:
Isn't it the same decorator for all the stories? If so, can't it be set as a global decorator?

Reply:
Why haven't I think of this?